appinventor/docs/reference/components/sensors.html

```diff
@@ -662,43 +662,68 @@ <h2 id="LocationSensor">LocationSensor</h2>
-  <dd></dd>
+  <dd><p>The LocationSensor will be able to locate the device with a varying degree of confidence, based on the quality of satellite, cell towers, and other data used to estimate location. The Accuracy value is the radius in meters around the sensor's detected location. The device is most likely located within this radius. More precise location detection will result in a smaller accuracy number, which allows the app to have more confidence where the device is actually located.</p>
+      <p>If the accuracy is not known, the return value is 0.0</p></dd>
   <dt><code><em>Altitude</em></code></dt>
-  <dd></dd>
+    <dd><p>Altitude of the Android device in meters, if available.</p>
+    <p>Altitude is measured from the <a target="_blank" href="https://gisgeography.com/wgs84-world-geodetic-system/">World Geodetic System 84 reference ellipsoid</a>,
+        not sea level. </p>
+    <p>Note that it is difficult for devices to accurately sense altitude. Altitude reported on a phone/tablet can
+        easily be off by 30 meters or more.</p></dd>
   <dt><code><em>AvailableProviders</em></code></dt>
-  <dd></dd>
+  <dd>List of available service providers, such as gps or network. This information is provided as a list and in text
+      form.</dd>
   <dt><code><em>CurrentAddress</em></code></dt>
-  <dd></dd>
+    <dd><p>Physical street address of the Android device from Google's map database.</p>
+    <p>The address might not always be available from the provider, and the address reported may not always be of the building where the device is located.</p>
+    <p>If Google has no address information available for a particular location, this will return No address available.</p>
+    </dd>
   <dt><code>DistanceInterval</code></dt>
-  <dd>Determines the minimum distance interval, in meters, that the sensor will try to use for sending out location updates. For example, if this is set to 5, then the sensor will fire a LocationChanged event only after 5 meters have been traversed. However, the sensor does not guarantee that an update will be received at exactly the distance interval. It may take more than 5 meters to fire an event, for instance.</dd>
+  <dd><p>Determines the minimum distance interval, in meters, that the sensor will try to use for sending out location
+      updates. For example, if this is set to 50, then the sensor will fire a LocationChanged event only after 50 meters
+      have been traversed. However, the sensor does not guarantee that an update will be received at exactly the
+      distance interval. It may take more than 5 meters to fire an event, for instance.</p>
+    <p>It is also useful to check against Accuracy when using this property. When your device is moving, the accuracy
+    of the detected location is constantly changing.</p></dd>
   <dt><code>Enabled</code></dt>
-  <dd></dd>
+  <dd>If true, the LocationSensor is reading data from the GPS. If false, it is not. This value does not affect whether
```

Comment:
I'm not sure/clear about the last sentence here. It can certainly have an effect on whether the GPS is on or off. If the GPS is off, and you enable the LocationSensor, the GPS receiver will turn on. Maybe this was intended to say that it doesn't change the whether the device's settings enable/disable GPS?

Reply:
The text here looks like it describes a passive property that merely indicates whether or not data can be read from the GPS.

Is that not an accurate description of the property?

Reply:
The `Enabled` property in App Inventor controls whether or not we are registered for location events. For example, this property can be `False`, but if you're running Google Maps in the background your GPS is probably still on. Likewise, this property can be `True`, but if the user had turned off location services then the app won't get any location data. So when the text says "This value does not affect whether the device's GPS is on or off" what it really means is whether location services are enabled/disabled. It only affects whether the app is going to attempt to request location data. It's more nuanced than what the current text implies.

Reply:
I have checked in a proposed change to the language here.